packages/cli/src/commands/deploy.ts

```diff
@@ -16,7 +40,9 @@ const runTasks = async (
 ): Promise<void> =>
   Script.init(`boost ${Brand.dangerize('deploy')} [${environment}] 🚀`, loader)
     //TODO: We should install dependencies in production mode before deploying
+    .step('Removing dev dependencies', pruneDependencies)
     .step('Deploying', (config) => deployer(config, logger))
+    .step('Reinstalling dependencies', reinstallDependencies)
```

Comment:
Here, you can pass `skipRestoreDependencies` as a boolean parameter to the `runTasks` function, and pass it to `reinstallDependencies` like so:
```suggestion
    .step('Reinstalling dependencies', reinstallDependencies.bind(null, skipRestoreDependencies))
``` 

The `bind` method receives a possible `this` as a first parameter (we don't use it so we pass `null`), and then the rest of the parameters in order, for the missing parameters it returns a new function. [Here's a much better explanation by @javiertoledo](https://dev.to/theagilemonkeys/currying-in-typescript-just-for-fun-5h9).

And finally, in the `run` method... (keep reading)